src/services/sdkService.ts

```diff
@@ -206,9 +205,13 @@ export default class SdkService {
       return this.walletService.getCustomToken(address);
     }
 
-    const tokenResponse = await this.sdk.tokenService.getTokens({ addresses: [{ addresses: [address], chainId }] });
+    const tokenResponse = await this.sdk.metadataService.getMetadataForChain({
```

Comment:
We can now fetch the name!! 🎉 

I marked it as required, is that ok? Other options would be "best effort" or "can ignore"